.github/workflows/e2e.yml

```diff
@@ -40,6 +40,7 @@ jobs:
       # Since the issue comment event runs against the main branch
       # we need to figure out a way to checkout the correct revision 
       # if we want to work with the changes from a pull request.
+      if: github.event.issue.pull_request && startsWith(github.event.comment.body, '/ci')
```

Comment:
Is it possible to perform this check once and set global variable?

Reply:
That's check is matter only in context of step

Reply:
It doesn't matter - we need to avoid code duplication when possible. Workflow is started in 2 cases:
1. Push in master branch
2. Comment `/ci` in PR
Two conditions can be checked at the beginning of workflow

Reply:
> `if` to prevent a job from running unless a condition is met. 

All `if` in steps handles `push` and `comment` events.

What is exactly do you suggest?

Reply:
I suggest to use variable like `is_pr_validation` and set it to true at the beginning of workflow when `github.event.issue.pull_request && startsWith(github.event.comment.body, '/ci')` is true and use this variable further instead of repeating `if: github.event.issue.pull_request && startsWith(github.event.comment.body, '/ci')` multiple times. Is that possible?

Reply:
I'm not sure if it's good. Your suggestion downgrade readability.

Reply:
@safronovD please add your thoughts on this